File: scripts/srv_server.py
#!/usr/bin/env python

import rospy 
from ros_tutor.srv import *

def srv_callback(req):
	rq_a = req.digit_1
	rq_b = req.digit_2
	rq_c = req.digit_3
	
	if rq_a == 2 and rq_b == 5 and rq_c == 5:
		return tutor_srvResponse("Accepted")
	else:
		return tutor_srvResponse("Deny")

def main():
	rospy.init_node("Service_ros")
	s = rospy.Service('password', tutor_srv, srv_callback)
	rospy.loginfo("Starting Service")

	rospy.spin()

if __name__ == "__main__":
	main()

File: ros_tutor/src/srv_server.cpp
#include "ros/ros.h"
#include "ros_tutor/tutor_srv.h"
#include <string>

bool srv_callback(ros_tutor::tutor_srv::Request  &req,
				  ros_tutor::tutor_srv::Response &res )
{
	// int result = req.digit_1 + req.digit_2 + req.digit_3;
	// res.status = std::to_string(result);
	// std::cout << result << std::endl;

	if (req.digit_1 == 2 and req.digit_2 == 5 and req.digit_3 == 5)
	{
		res.status = "Accepted";
	}
	else{
		res.status = "Deny";
	}
	return true;
}

int main(int argc, char **argv)
{
	ros::init(argc, argv, "Service_ros");
	ros::NodeHandle n;

	ros::ServiceServer service = n.advertiseService("password", srv_callback);

	ros::spin();

	return 0;
}
